tests/transforms/test_qcut.py

```diff
@@ -936,6 +947,272 @@ def test_multiple_conversions(self):
+
+        g = qcut.tape_to_graph(tape)
+        qcut.replace_wire_cut_nodes(g)
+        subgraphs, communication_graph = qcut.fragment_graph(g)
+        tapes = [qcut.graph_to_tape(sg) for sg in subgraphs]
+
+        fragment_configurations = [qcut.expand_fragment_tapes(tape) for tape in tapes]
+        frag_tapes_meas = fragment_configurations[0][0]
+        frag_tapes_prep = fragment_configurations[1][0]
+
+        assert len(frag_tapes_meas) == 3
+        assert len(frag_tapes_prep) == 4
+
+        frag_meas_ops = [
+            qml.RX(0.432, wires=[0]),
+            qml.RY(0.543, wires=[1]),
+            qml.CNOT(wires=[0, 1]),
+            qml.RZ(0.24, wires=[0]),
+            qml.RZ(0.133, wires=[1]),
+        ]
+
+        with qml.tape.QuantumTape() as tape_00:
+            for op in frag_meas_ops:
+                qml.apply(op)
+            qml.expval(qml.PauliZ(wires=[0]) @ qml.Identity(wires=[1]))
+            qml.expval(qml.PauliZ(wires=[0]) @ qml.PauliZ(wires=[1]))
+
+        with qml.tape.QuantumTape() as tape_01:
+            for op in frag_meas_ops:
+                qml.apply(op)
+            qml.expval(qml.PauliZ(wires=[0]) @ qml.PauliX(wires=[1]))
+
+        with qml.tape.QuantumTape() as tape_02:
+            for op in frag_meas_ops:
+                qml.apply(op)
+            qml.expval(qml.PauliZ(wires=[0]) @ qml.PauliY(wires=[1]))
+
+        frag_meas_expected_tapes = [tape_00, tape_01, tape_02]
+
+        frag_prep_ops = [qml.CNOT(wires=[1, 2]), qml.RX(0.432, wires=[1]), qml.RY(0.543, wires=[2])]
+
+        with qml.tape.QuantumTape() as tape_10:
+            qml.Identity(wires=[1])
+            for op in frag_prep_ops:
+                qml.apply(op)
+            qml.expval(qml.Identity(wires=[1]))
+
+        with qml.tape.QuantumTape() as tape_11:
+            qml.PauliX(wires=[1])
+            for op in frag_prep_ops:
+                qml.apply(op)
+            qml.expval(qml.Identity(wires=[1]))
+
+        with qml.tape.QuantumTape() as tape_12:
+            qml.Hadamard(wires=[1])
+            for op in frag_prep_ops:
+                qml.apply(op)
+            qml.expval(qml.Identity(wires=[1]))
+
+        with qml.tape.QuantumTape() as tape_13:
+            qml.Hadamard(wires=[1])
+            qml.S(wires=[1])
+            for op in frag_prep_ops:
+                qml.apply(op)
+            qml.expval(qml.Identity(wires=[1]))
+
+        frag_prep_expected_tapes = [tape_10, tape_11, tape_12, tape_13]
+
+        for tape_meas, exp_tape_meas in zip(frag_tapes_meas, frag_meas_expected_tapes):
+            compare_tapes(tape_meas, exp_tape_meas)
+
+        for tape_prep, exp_tape_1 in zip(frag_tapes_prep, frag_prep_expected_tapes):
+            compare_tapes(tape_prep, exp_tape_1)
+
+    def test_multi_qubit_expansion_measurements(self):
+        """
+        Tests that a circuit with multiple MeasureNodes on a single wire gives
+        the correct measurements after expansion
```

Comment:
The circuit below doesn't have multiple `MeasureNodes` on the same wire :thinking: